apix-integrationtests/alfresco/src/main/java/eu/xenit/apix/tests/JavaApiBaseTest.java

```diff
@@ -108,6 +87,16 @@ protected boolean removeTestNode(NodeRef nodeRef) {
         return success;
     }
 
+    protected boolean removeTestPersonNode(NodeRef nodeRef) {
+        NodeService alfrescoNodeService = serviceRegistry.getNodeService();
+        boolean success = false;
+        if (alfrescoNodeService.exists(nodeRef)) {
+            alfrescoNodeService.deleteNode(nodeRef);
+            success = true;
```

Comment:
no check to see if the node was deleted afterwards?

Reply:
This was temporarely added to try and create a personNode for the AssociationsTest. Since the used ASSOC_TYPE was'nt allowed anymore between 2 files/folders. Which crashed the test. I ended up just changing the ASSOC_TYPE

Reply:
deleted.